src/app/assessment/assessment.service.ts

```diff
@@ -408,20 +423,25 @@ export class AssessmentService {
     return answer;
   }
 
-  saveAnswers(assessment, answers, action) {
+  saveAnswers(params: SaveAnswersParams) {
```

Comment:
We don't need to create this parameters as an Interface, we can just put `submissionId` as optional parameter

Reply:
BTW, `AssessmentSubmission`, `AssessmentReview` these are all names that we used in Practera backend for naming convention. Here is App we can just use `assessment`, `submission`, `review` for short